webpack.client.js

```diff
@@ -47,7 +53,10 @@ module.exports = {
   },
   plugins: [
     new HtmlWebPackPlugin({
-      template: "./src/homepage/index.html",
+      templateParameters: {
+        googleTag: JSON.parse(config).googleTag,
```

Comment:
This only works if there is a `config.json` file available and throws `TypeError: Cannot read property 'googleTag' of null` if file is not there.

You should probably check if `config` is an object it is before reading `googleTag` property of `config` object.